_overviews/jdk-compatibility/overview.md

```diff
@@ -48,7 +50,7 @@ In almost every case, you're free to use the JDK and JVM of your choice.
 
 JDK 8 users typically use the Oracle JDK or some flavor of OpenJDK.
 
-Most JDK 11+ users are using either OpenJDK or GraalVM.
+Most JDK 11+ users are using either OpenJDK or [GraalVM](https://www.graalvm.org/docs/reference-manual/scala/) which runs in the context of OpenJDK. Developers can use [sbt-native-image](https://github.com/scalameta/sbt-native-image) which uses GraalVM under the hood.
```

Comment:
This needs clarification. Are you referring to the GraalVM compiler which using JVMCI can be utilized as a C2 compiler by OpenJDK?

Reply:
To be very frank, I have not gone through the source code of `sbt-native-image`. I just read that it uses GraalVm under the hood. If this line is confusing, I can completely remove it. Else, if you would like to have an alternate explanation, I would be happy to put it.
Thanks

Reply:
`sbt-native-image` mention deserves a separate section ( and you can add sbt-native-packager https://www.scala-sbt.org/sbt-native-packager/formats/graalvm-native-image.html )

We can mention that there are several known projects already compiling with the GraalVM AOT compiler and therefore running on SubstrateVM.

@chaitanyawaikar this is always the confusion with Graal:

 - AOT compiler
 - JIT compiler (as opposed to C2 in a normal JVM)
 - Polyglot runtime
 
 I think your addition covers pretty well the JIT part!

Reply:
@andreaTP does your comment mean that my changes look good ? Sorry, but I am still confused :(

Reply:
ping @andreaTP

Reply:
sorry, missed the comment, not really for this PR I suggest that @chaitanyawaikar simply remove the line:

> Developers can use [sbt-native-image](https://github.com/scalameta/sbt-native-image) which uses GraalVM under the hood.

and I will follow up with the changes for `native-image`, works for you @chaitanyawaikar ?

Reply:
Thanks for the clarification @andreaTP . Removed the line